lib/routes/lnurl/payment/lnurl_payment_handler.dart

```diff
@@ -48,9 +48,9 @@ Future<LNURLPageResult?> handlePayRequest(
       isLnurlPayment: true,
       firstPaymentItemKey: firstPaymentItemKey,
       paymentFunc: () => context.read<AccountBloc>().lnurlPay(
-            amount: paymentInfo!.amount,
+            amountMsat: paymentInfo!.amount * 1000,
             comment: paymentInfo.comment,
-            reqData: requestData,
+            data: data,
           ),
```

Comment:
Let's change `lnurlPay` on `AccountBloc` to take `LnUrlPayRequest` as a param and convert this to a block body:

 ```
paymentFunc: () {
     final accBloc = context.read<AccountBloc>();
     final req = sdk.LnUrlPayRequest(data: data, amountMsat: paymentInfo!.amount * 1000, comment: paymentInfo.comment);
     return accBloc.lnurlPay(req:req);
 },
```

Reply:
added